safe/gui/tools/wizard/test/test_keyword_wizard.py

```diff
@@ -1031,14 +1024,14 @@ def test_integrated_line(self):
         # check if in step subcategory
         self.check_current_step(dialog.step_kw_subcategory)
 
-        expected_subcategories = ['Road']
+        expected_subcategories = ['Roads']
```

Comment:
It will be less maintenance if we change the value again in the definitions in the future. The main purpose of these tests is to check the wizard.